contributor-docs/component-contents-api-patterns.md

```diff
@@ -2,7 +2,7 @@
 
 Consider a React component that renders a list of items. Here are two possible APIs that component might expose, both achieving an equivalent result.
 
-### A: Contents passed as React children
+### A: Contents passed as React children <!-- markdownlint-disable-line heading-increment -->
```

Comment:
@primer/react-reviewers This is a TODO! I'm not sure what the heading structure should be. Feel free to suggest something in this PR to fix this!